src/main/kotlin/net/irisshaders/lilybot/utils/DatabaseHelper.kt

```diff
@@ -274,6 +274,16 @@ object DatabaseHelper {
 		collection.insertOne(ThreadData(inputThreadId, newOwnerId))
 	}
 
+	/**
+	 * This function deletes the ownership data stored in the database for the given [threadId].
+	 *
+	 * @param threadId The ID of the thread to delete
+	 */
+	suspend fun removeThread(threadId: Snowflake) {
```

Comment:
If I'm being nitpicky this should be called deleteThread for consistency with other functions. 

In addition, I'd suggest renaming the threadId input to inputThreadId for clarity with the database filters.